docs/install/upgrade-installation-with-operator.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+The upgrade is relatively more straightforward, if you install Knative with the Knative Operator. We support upgrading
+by a single [minor](https://semver.org/) version number. For example, if you have v0.17 installed, you must upgrade to
+v0.18 before attempting to upgrade to v0.19. The attribute `spec.version` is the only field you need to change in the
+Serving or Eventing CR to perform an upgrade. You do not need to specify the version in terms of the `patch` number,
+because the Knative Operator will match the latest available `patch` number, as long as you specify `major.minor` for
+the version. For example, you only need to specify `0.19` to upgrade to the latest v0.19 release. There is no need to
+know the exact `patch` number.
+
+The Knative Operator implements a minus 3 principle to support the Knative versions, which means the current version
+of the Operator can support Knative with the version back 3 in terms of the `minor` number. For example, if the
+current version of the Operator is 0.19.x, it bundles and supports the installation of Knative with the versions,
+0.16.x, 0.17.x, 0.18.x and 0.19.x.
```

Comment:
Can I upgrade directly from 0.17 ... to 0.19 ? 

If no: let's add a note here

Reply:
Check line 7: 
```
The upgrade is relatively more straightforward, if you install Knative with the Knative Operator. We support upgrading by a single [minor](https://semver.org/) version number. For example, if you have v0.17 installed, you must upgrade to v0.18 before attempting to upgrade to v0.19.
```